test_TicTacToe.py

```diff
@@ -0,0 +1,261 @@
+import TicTacToe
+import pytest
+from string import printable
```

Comment:
To make it clear in further code could this be imported as something more verbose, like printable_chars.
So it would be from string import printable as printable_chars.

Reply:
Resolved in 8368ed9